### DISCUSSION TITLE
Ipopt master branch

### DISCUSSION THREAD

**Discussion by bradbell** [coin-or/Ipopt#494] [Q&A] [answered]
It appears that Ipopt no longer has a master branch ?

**Comment by svigerske** [comment, marked as answer]
Indeed. There was no use for it, so it got removed.

**Reply by bradbell** [reply]
So where does Ipopt put new features once they are debugged and sure to be in the next stable branch ?

**Reply by svigerske** [reply]
Either the latest stable/x.y branch or branch devel.
I added some info to the README recently: https://github.com/coin-or/Ipopt#versioning
